src/main.cpp

```diff
@@ -515,7 +518,7 @@ void setup() {
     SensirionSGP30::Setup();
     HX711::Setup();
 #endif
-    xTaskCreatePinnedToCore(scanTask, "scanTask", SCAN_TASK_STACK_SIZE, nullptr, 1, &scanTaskHandle, CONFIG_BT_NIMBLE_PINNED_TO_CORE);
+    xTaskCreatePinnedToCore(scanTask, "scanTask", 7000, nullptr, 1, &scanTaskHandle, CONFIG_BT_NIMBLE_PINNED_TO_CORE);
```

Comment:
Why?

Reply:
Sorry forgot to changing that, is it okay to increase the scans stack size? My code is using to much stack and running it on a higher stack size went fine.

Reply:
Yeah, we can increase it if it's truly needed.  If might be a better option to try and use the heap more instead since we're very memory limited.

Reply:
But the tasks stack is allocated on the heap right? Aren't they sharing the same memory or am i understanding something wrong here? ([url](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-reference/system/mem_alloc.html))